src/docs/product/performance/web-vitals.mdx

```diff
@@ -35,18 +34,22 @@ If you notice a region of interest on any of the histograms, click and drag over
 
 If you wish to see all of the data available, open the dropdown and click "View All". You will likely see extreme outliers when you click "View All". You can click and drag over an area to zoom in for a more detailed view.
 
-## Web Vitals
+## Thresholds
 
-[Web Vitals](https://web.dev/vitals/) are a set of metrics defined by Google to measure render time, response time, and the sum of shifting rendering processes. Each data point provides insights about the overall performance of your application.
+Google's "Bad", "Needs improvement", and "Good" levels are used to classify data points into red, yellow, and green for the corresponding Web Vitals.
```

Comment:
actually we kinda don't since https://github.com/getsentry/sentry/pull/23839. there might not be enough room in some places to rename them all to "needs improvement" though, so I guess I can explain that sometimes we call "needs improvement" "meh".